sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerClient.cs

```diff
@@ -19,55 +18,44 @@ public partial class ConfidentialLedgerClient
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="options"> The options for configuring the client. </param>
         public ConfidentialLedgerClient(Uri ledgerUri, TokenCredential credential, ConfidentialLedgerClientOptions options)
-        {
-            if (ledgerUri == null)
-            {
-                throw new ArgumentNullException(nameof(ledgerUri));
-            }
-            if (credential == null)
-            {
-                throw new ArgumentNullException(nameof(credential));
-            }
-
-            var actualOptions = options ?? new ConfidentialLedgerClientOptions();
-            var transportOptions = GetIdentityServerTlsCertAndTrust(ledgerUri, actualOptions);
-            ClientDiagnostics = new ClientDiagnostics(actualOptions, true);
-            _tokenCredential = credential;
-            var authPolicy = new BearerTokenAuthenticationPolicy(_tokenCredential, AuthorizationScopes);
-            _pipeline = HttpPipelineBuilder.Build(
-                actualOptions,
-                Array.Empty<HttpPipelinePolicy>(),
-                new HttpPipelinePolicy[] { authPolicy },
-                transportOptions,
-                new ResponseClassifier());
-            _ledgerUri = ledgerUri;
-            _apiVersion = actualOptions.Version;
-        }
+            : this(ledgerUri, credential: credential, options: options, identityServiceClient: default)
```

Comment:
Nit: You use `default` here but `null` below, and neither is necessary since it defaults to `null` anyway. Perhaps just elide?